hack/make-rules/test-cmd-util.sh

```diff
@@ -237,9 +271,14 @@ setup() {
 # Kubectl version (--short, --client, --output) #
 ########################################################
 run_kubectl_version_tests() {
+  set -o nounset
```

Comment:
we need to not fail the test if the sh2ju.sh script exits non-0 because that is expected for failed tests.  do you know of a better way of doing this?